### ISSUE TITLE
update script issue ?

### ISSUE THREAD

**Issue by zigzagsui** [mrv777/sui-installation-scripts#2]
Hello, after i updated node from 0.6.2 to 0.6.3 with the update-script,  if i relaunch the update script when it's done , it still says
[INFO] Checking node version
[INFO] Local node version: 0.6.2
[INFO] Remote node version: 0.6.3

**Comment by ViSTiE** [comment]
Yeah, I've the same issue. When I check my json-rpc, my version is on 0.6.1.

**Comment by mrv777** [comment]
> Hello, after i updated node from 0.6.2 to 0.6.3 with the update-script, if i relaunch the update script when it's done , it still says [INFO] Checking node version [INFO] Local node version: 0.6.2 [INFO] Remote node version: 0.6.3

Sorry for the confusion.  The VERSION file also needs to be downloaded and I will work on fixing that in the script

Oddly the SUI team did not make a tag for 0.6.4 here: https://github.com/MystenLabs/sui/tags
So the script thinks 0.6.3 is still the newest.  I'm not sure why they did this

**Comment by zigzagsui** [comment]
yes, i had the VERSION file too, with a 4 in it. it still saying it needs to update.

**Comment by ViSTiE** [comment]
@mrv777 Is it possible to grab the latest Sui node version from a different source? The tag for 0.6.4 is still unavailable yet, but the version is out according to the Discord node-operators channel. 
Even after executing the update script this morning again, my Cargo.toml file still says I'm on 0.6.2.

**Comment by mrv777** [comment]
@ViSTiE I think that was just a missed version bump in that file as my understanding is that even if you upgrade to 0.6.4, cargo.toml still shows 0.6.2
